build.gradle

```diff
@@ -893,8 +892,8 @@ project(':core') {
     from(project(':streams:test-utils').configurations.runtime) { into("libs/") }
     from(project(':streams:examples').jar) { into("libs/") }
     from(project(':streams:examples').configurations.runtime) { into("libs/") }
-    from(project(':remote-storage-managers:hdfs').jar) { into("libs/") }
-    from(project(':remote-storage-managers:hdfs').configurations.runtime) { into("libs/") }
+    from(project(':remote-storage-managers:hdfs').jar) { into("remote-storage/hdfs/libs/") }
```

Comment:
@ying-zheng should we put this in external/hdfs/lib

Reply:
I think we will anyway move hdfs rsm to a different repo. This is just temporary.

Reply:
changed the path to external